rfc6962-bis.xml

```diff
@@ -1602,13 +1602,7 @@ corrective action when a misissue is detected.
       </section>
       <section title="Redaction of Public Domain Name Labels">
         <t>
-          CAs SHOULD NOT redact domain name labels in precertificates such that the entirety of the domain space below the unredacted part of the domain name is not owned or controlled by a single entity
-(e.g. <spanx style="verb">?.com</spanx> and
-<spanx style="verb">?.co.uk</spanx> would both be problematic). Logs
-MUST NOT reject any precertificate that is overly redacted but which is
-otherwise considered compliant. It is expected that monitors will treat overly
-redacted precertificates as potentially misissued. TLS clients MAY reject a
-certificate whose corresponding precertificate would be overly redacted, perhaps using the same mechanism for determining whether a wildcard in a domain name of a certificate is too broad.
+          Redaction of domain name labels carries the same risks as the use of wildcards (See Section 7.2 of <xref target="RFC6125"/>, for example). CAs should take care not to redact domain name labels in precertificates too broadly, such that the entirety of the domain space below the unredacted part of the domain name may be controlled by multiple entities, such as domains in the <xref target="Public.Suffix.List">Public Suffix List</xref> (e.g. <spanx style="verb">?.com</spanx> and <spanx style="verb">?.co.uk</spanx> would both be problematic). Logs MUST NOT reject any precertificate that is overly redacted but which is otherwise considered compliant. It is expected that monitors will treat overly redacted precertificates as potentially misissued. TLS clients MAY reject a certificate whose corresponding precertificate would be overly redacted, perhaps using the same mechanism for determining whether a wildcard in a domain name of a certificate is too broad.
```

Comment:
Shall we also update the "overly redacted" language in this paragraph for logs, monitors and TLS clients?  If "overly redacted" is somewhat subjective for CAs, then it follows that it's somewhat subjective for logs, monitors and TLS clients too.  Something like this...
"Logs MUST NOT reject any precertificate that could be considered to be overly redacted. It is expected that monitors will treat precertificates that they consider to be overly redacted as potentially misissued. TLS clients MAY reject a certificate if they consider the corresponding precertificate to be overly redacted..."